editor/editor_help_search.cpp

```diff
@@ -40,6 +40,7 @@ void EditorHelpSearch::_update_icons() {
 	search_box->set_right_icon(results_tree->get_theme_icon(SNAME("Search"), SNAME("EditorIcons")));
 	search_box->set_clear_button_enabled(true);
 	search_box->add_theme_icon_override("right_icon", results_tree->get_theme_icon(SNAME("Search"), SNAME("EditorIcons")));
+	keywords_button->set_icon(results_tree->get_theme_icon(SNAME("AnimationTrackList"), SNAME("EditorIcons")));
```

Comment:
Might need a better choice of icon, since it is visually very similar to the "Show Hierarchy" icon.